index.html

```diff
@@ -39,10 +39,9 @@
             <p class="title">-INTRODUCTION</p>
             <p class="hi">Hi There!👋</p>
             <h1 class="my-name">I'm <span>Divya Kumari</span></h1>
-            <p class="slogun">I help startup to create stunning websites,
-              that make impact on market.</p>
+            <p class="slogun">I empower startups to craft compelling websites that leave a lasting impression on the market.</p>
```

Comment:
that's not a grammar mistake. you changed words. I would like to keep the previous one.

Reply:
The grammatical error in the sentence is the incorrect placement of "to." It should be "I help startups create stunning websites that make an impact on the market." , but sure i will leave it as is